prow/scripts/lib/utils.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+utils::checkRequiredVars() {
```

Comment:
Please change the function name to snake_case to preserve consistency

Reply:
Function names changed to snake_case